File: SimpleHabit/app/src/main/java/com/me/simplehabit/network/responses/GetCurrentProgramResponse.java
package com.me.simplehabit.network.responses;

import com.google.gson.annotations.SerializedName;
import com.me.simplehabit.data.vos.CurrentProgramVO;

public class GetCurrentProgramResponse extends BaseResponse{

    @SerializedName("currentProgram")
    private CurrentProgramVO currentProgram;

    public CurrentProgramVO getCurrentProgram() {
        return currentProgram;
    }

    public void setCurrentProgram(CurrentProgramVO currentProgram) {
        this.currentProgram = currentProgram;
    }
}

File: SimpleHabit/app/src/main/java/com/me/simplehabit/network/responses/GetCurrentProgramResponse.kt
package com.me.simplehabit.network.responses

import com.google.gson.annotations.SerializedName
import com.me.simplehabit.data.vos.CurrentProgramVO

data class GetCurrentProgramResponse(

        @SerializedName("currentProgram")
        val currentProgram: CurrentProgramVO

) : BaseResponse()
